tests/pplat.cu

```diff
@@ -209,25 +240,32 @@ int main(int argc, char *argv[])
         pp_kernel<<< 1, 1 >>>((uint32_t *)d_buf_cuptr, (uint32_t *)h_buf_cuptr, num_iters);
 
         uint32_t i = 1;
-        while (READ_ONCE(*h_buf) != i) ;
+        // Wait for pp_kernel to be ready before starting the time measurement.
+        clock_gettime(MYCLOCK, &beg);
+        while (READ_ONCE(*h_buf) != i) {
+            check_timeout(beg, timeout_us);
```

Comment:
I would check for timeout every 10^3 iterations or so

Reply:
so that you can turn it on by default with a large value, say 1 second, without affecting the latency measurement

Reply:
Implemented.